chromadb/db/mixins/sysdb.py

```diff
@@ -342,7 +342,11 @@ def get_segments(
                 rows = list(segment_rows)
                 type = str(rows[0][1])
                 scope = SegmentScope(str(rows[0][2]))
-                collection = self.uuid_from_db(rows[0][3]) if rows[0][3] else None
+
+                if not rows[0][3]:
+                    raise ValueError("Segment is missing collection ID")
```

Comment:
assuming this is a data corruption or bad case?

Reply:
yep, should never happen but technically this field is nullable in the table

Reply:
given that the number of segments is small I would prefer a migration that copied the data and altered the schema to not nullable - wdyt?

Reply:
On distributed same thing?